Gruntfile.js

```diff
@@ -488,34 +398,33 @@ module.exports = function(grunt) {
     'build-admin',
     'build-config',
     'copy-static-files-to-api',
-    'copy:api-ddocs',
+    'exec:copy-api-ddocs',
   ]);
 
   grunt.registerTask('copy-static-files-to-api', 'Copy build files and static files to api', [
-    'copy:api-bowser',
-    'copy:api-resources',
-    'copy:built-resources',
-    'copy:webapp-static',
-    'copy:admin-static',
+    'exec:copy-api-bowser',
+    'exec:copy-api-resources',
+    'exec:copy-webapp-static',
```

Comment:
Do we no longer need to copy all built resources?

Reply:
I... don't remember.

The command just moved everything from `build/static` to `api/build/static`. I've hunted around and it looks like everything now just compiles directly to `api/build/static` so it shouldn't be needed any more.

I haven't confirmed it 100% but I'd be shocked if it some static file was missed but the e2e tests still pass.

Reply:
This is quite possible. Thanks for checking!